### PR TITLE
:sparkles: Convert docsy to a hugo module

### PR DESCRIPTION
**Pull request by mauromorales**
When working on the changes for the homepage we had a couple of issues because of the git submodule. This PR changes docsy to use the new hugo modules for installation of themes and updates the docsy theme to the latest stable available.

### CODE REVIEW

**Comment by mauromorales** [comment]
@mudler whenever you get a chance, could you give me access to the config of the Netlify? or add the following env var: `HUGO_VERSION` to be set to `0.110.0`

**Comment by mauromorales** [comment]
Navbar is broken on homepage, other than that seems good

image

File: Earthfile [hunk 1]
```diff
@@ -19,7 +21,6 @@ docs:
     WORKDIR ./docs
 
     RUN npm install postcss-cli
-    RUN npm run prepare
```
Comment (Itxaka) [inline Earthfile:22]: oh shit, this is gonna improve the building of the arm64 packages by a lot!

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module github.com/kairos-io/kairos-docs
+
+go 1.20
+
+require (
+	github.com/google/docsy v0.7.1 // indirect
```
Comment (Itxaka) [inline go.mod:6]: :eyes: ?

File: Earthfile (hunk 1 above)
Comment (mauromorales) [reply Earthfile:22]: maybe I should test that and make sure I didn't break anything there... sounds too good to be  true :P

**Review by Itxaka — APPROVED** [review]
looks good, I cant find any issues in the preview!

**Pull request merged**